client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx

```diff
@@ -153,14 +156,17 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 			<>
 				<p>
 					{ translate(
-						'%(emails)s are almost ready! Complete the setup to activate your new email addresses.',
+						'Your emails {{strong}}%(emails)s{{/strong}} are almost ready! Complete the setup to activate your new email addresses.',
 						{
 							args: {
 								emails: emails,
 							},
 							comment:
 								'%(emails)s will be a list of email addresses separated by a comma, ' +
 								'e.g. test@example.com, test2@example.com',
+							components: {
+								strong: <strong />,
+							},
```

Comment:
This is unrelated to your changes but we're not handling [pluralization](https://github.com/Automattic/wp-calypso/blob/master/packages/i18n-calypso/README.md#pluralization) correctly here. While English has one singular and one plural form, this is indeed not the case for many other languages that may have complex rules for forming plurals (see PCYsg-3hI-p2#beware-plurals for an example). This can be addressed in another pull request though, especially if there are other notice components which follow the wrong approach.

Reply:
Nice catch - we definitely need to fix this.